includes/definitions.inc.php

```diff
@@ -595,6 +595,19 @@
 $config['os'][$os]['over'][4]['text']  = 'Number of Clients';
 $config['os'][$os]['icon']             = 'cisco';
 
+$os = 'ciscoons';
```

Comment:
I think this should be `ons`. Just like `ios` and the others are all cisco based but don't use cisco in the os name.